feature-libs/organization/administration/styles/_buttons.scss

```diff
@@ -47,6 +47,7 @@
     }
 
     padding: var(--cx-spatial-base);
+    margin: var(--cx-spatial-base);
```

Comment:
The ticket is about b2b unit page. The styling change is quite global (affects all organization pages). So it likely can break layouts containing buttons in other organization pages. Would you mind double checking other pages didn't change?
To test it, you can for example:
- have 2 locally cloned repositories of Spartacus (please mind to run `yarn` after clone)
- on one of them checkout to develop branch; on the another to your branch
- `yarn start --port 4200` on develop. and `yarn start pp-port 4201` on your branch
- in browser open 2 tabs, i.e. one localhost:4200 and another localhost:4201. 
- browse in depth the organization pages and forms, and switch tabs to compare visually if nothing changed, except the b2b unit page that you try to fix ;)

Reply:
Hi Krzysztof，

Thanks a lot for your detailed comment. Yes I agree, some more manual tests are necessary to avoid any side effect. I will do those tests and update you once done.

Best Regards,
Jerry

Reply:
> The ticket is about b2b unit page. The styling change is quite global (affects all organization pages). So it likely can break layouts containing buttons in other organization pages. Would you mind double checking other pages didn't change?
> To test it, you can for example:
> 
> * have 2 locally cloned repositories of Spartacus (please mind to run `yarn` after clone)
> * on one of them checkout to develop branch; on the another to your branch
> * `yarn start --port 4200` on develop. and `yarn start -port 4201` on your branch
> * in browser open 2 tabs, i.e. one localhost:4200 and another localhost:4201.
> * in both tabs login as linda.wolf
> * simultaneously browse in depth the organization pages and forms, and switch tabs to compare visually if something changed, except the b2b unit page that you aim to fix ;)

Hi Krzysztof,

I have done manual test according to your suggestion. After margin is introduced for all elements in org page with css selector **button.button** and **a.button**, the pages which contain such elements have a very slight difference compared with previous one. I have uploaded some screenshot about "between change VS after change" into this link: https://github.com/wangzixi-diablo/Spartacus-learning/issues/5

In my opinion, the manual test proves that no side effect is introduced. Would you please kindly check and approve? Thanks a lot for your help!

Best Regards,
Jerry